Example/HorizonCalendarExample/HorizonCalendarExample/Demo View Controllers/DemoViewController.swift

```diff
@@ -57,18 +57,17 @@ class DemoViewController: UIViewController {
         calendarView.trailingAnchor.constraint(
           lessThanOrEqualTo: view.layoutMarginsGuide.trailingAnchor),
         calendarView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-        calendarView.widthAnchor.constraint(lessThanOrEqualToConstant: 375)
+        calendarView.widthAnchor.constraint(lessThanOrEqualToConstant: 375),
+        calendarView.widthAnchor.constraint(equalToConstant: 375).prioritize(at: .defaultLow),
```

Comment:
had to add this since `CalendarView` does _not_ have an intrinsic width, so the horizontal width is ambiguous without it.